cpp/autosar/test/rules/A7-1-2/test.cpp

```diff
@@ -127,7 +127,7 @@ class MissingConstexprClass {
   MissingConstexprClass(int i) = delete;           // NON_COMPLIANT
   MissingConstexprClass(int i, LiteralClass lc) {} // NON_COMPLIANT
 private:
-  int m1 = 0; // COMPLIANT - non-static member variable
+  int m1 = 0;
```

Comment:
Needs "NON_COMPLIANT" marker

Reply:
Fixed in [e9b3ebd](https://github.com/github/codeql-coding-standards/pull/794/commits/e9b3ebd1c9e4e92d83ac47a6a65f119709e0a5a5)